ast/desugar/Desugar.cc

```diff
@@ -1107,6 +1107,19 @@ TreePtr node2TreeImpl(DesugarContext dctx, unique_ptr<parser::Node> what) {
                 }
             },
             [&](parser::CSend *csend) {
+                auto chainLvarName = core::NameRef::noName();
+                auto *chainTemp = csend;
+                while (chainTemp != nullptr) {
+                    if (chainTemp != csend) {
```

Comment:
Why only set `chainLvarName` when `chainTemp != csend`? Is that to avoid adding a use of `T.must` for cases like `x&.foo(x.bar)`? If so, would you mind adding a comment to that effect?